### DISCUSSION TITLE
Tero Subtitler | Manual

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#71] [General]
A manual was produced for Subtitle Workshop, but it will need updating for Tero Subtitler.

The desired format is HTML, but is anyone interested in helping? A possible editor to use is [BlueGriffon](http://bluegriffon.org), but other suggestions are welcome (cloud-based solutions are preferred).

**Comment by chenlung** [comment]
We should probably include some provision in relation to a file format's specification (from subtitles to shot changes)? Some will be obsolete and may not even have a website, so it's possible we'll have to produce something ourselves.

**Comment by GitGianluc** [comment]
> A manual was produced for Subtitle Workshop, but it will need updating for Tero Subtitler.
>
> The desired format is HTML, but is anyone interested in helping?

I would be glad to consult that manual! I believe it would be very helpful for translating the interface more effectively and providing a good localized version. However, I'm not confident in my English proficiency to contribute with new content... On the other hand, I do have a good understanding of HTML!

> A possible editor to use is [BlueGriffon](http://bluegriffon.org), but other suggestions are welcome (cloud-based solutions are preferred).

I'm not an expert, but why not consider building documentation pages directly on [GitHub and Docsify](https://wise4rmgodadmob.medium.com/how-to-create-a-documentation-site-with-docsify-and-github-pages-doc-as-a-code-c419c44e99b7)?

**Reply by chenlung** [reply]
I'm a native English speaker, so at least we have that base covered! If someone can improve on my efforts, I'm happy to take a backseat.

We could wait a few days for opinions from others, and if there are no objections, we can try your suggestion. I like the collaborative nature of Google Workspace, so even to make a draft that can be worked on by multiple people simultaneously would be great; I don't know if such a thing exists.

**Reply by GitGianluc** [reply]
> I'm a native English speaker

Oh, so you'll have to be lenient with me! 😆 


> We could wait a few days for opinions from others, and if there are no objections, we can try your suggestion.

OK, let's wait and see.

> I like the collaborative nature of Google Workspace, so even to make a draft that can be worked on by multiple people simultaneously would be great; I don't know if such a thing exists.

Yes, it would, but unfortunately I don't know either.

**Reply by chenlung** [reply]
> Yes, it would, but unfortunately I don't know either.

There appears to be third-party services offering this, but seemingly no native support from Google themselves.

**Comment by Henrysub** [comment]
Dear all, I've been asked to teach a subtitling workshop for students of a film school. I personally use a paid subtitling program, but am looking for an open source option that runs on Mac, since apparently all the film students work on mac. Tero subtitler seems a great option. I am wondering if someone has created a manual by now.  If yes, please let me know where to find it. I greatly appreciate any help.

**Comment by Henrysub** [comment]
Hi there, I wrote a small manual for the students. I'm sure there is more to the programme, but it might be useful for someone starting to use it for the first time.
[Tero Subtitler Quick User Manual.docx](https://github.com/user-attachments/files/23193431/Tero.Subtitler.Quick.User.Manual.docx)

**Reply by chenlung** [reply]
Thank you!

A fully fledged manual is still something to be ironed out.
